devfile.yaml

```diff
@@ -1,18 +1,25 @@
 schemaVersion: 2.1.0
 metadata:
   name: jboss-eap-quickstarts
-attributes:
-  che-theia.eclipse.org/sidecar-policy: mergeImage
 components:
   - name: tools
     container:
-      image: registry.redhat.io/jboss-eap-7/eap74-openjdk8-openshift-rhel7:7.4.0
+      image: quay.io/devspaces/udi-rhel8:3.1
+      memoryLimit: 3Gi
+      volumeMounts:
+        - name: m2
+          path: /home/user/.m2
+  - name: eap
+    container:
+      image: registry.redhat.io/jboss-eap-7/eap74-openjdk11-openshift-rhel8:7.4.4
```

Comment:
do we want this to just be floating tag 7.4 ?

Reply:
oh, I see they only have :latest ... never mind

https://catalog.redhat.com/software/containers/jboss-eap-7/eap74-openjdk11-openshift-rhel8/6054ceca93acb006e7349a98